env/iasConfig.go

```diff
@@ -41,7 +55,7 @@ type Identity struct {
 }
 
 // GetIASConfig parses the IAS config from the applications environment
-func GetIASConfig() (*Identity, error) {
+func GetIASConfig() (*DefaultIdentity, error) {
```

Comment:
What do you think about renaming this func to sth like `ReadIdentityConfig`?
The abbreviation ias should not be used externally as I now learned.

Reply:
ok, let me do that in a separate PR, i will also try to change the return type

Reply:
Okay for me.

Return type however should be ok.